Makefile

```diff
@@ -55,6 +55,10 @@ bin/brogue.exe: $(objects) windows/icon.o
 clean:
 	$(RM) src/brogue/*.o src/platform/*.o bin/brogue{,.exe}
 
+install:
+	install -m 557 bin/brogue $(DESTDIR)/usr/bin
+	install -d $(DESTDIR)/usr/share/assets
+	install bin/assets/* -t $(DESTDIR)/usr/share/assets
```

Comment:
I'd prefer not adding this; while it is easy for packagers to accomplish (with `DATADIR` and `linux/brogue-multiuser.sh`), system-wide Brogue on Linux isn't supported by default and I don't want to mislead people.

P.S. mode 557 gives write access to all users, is this intentional?

Reply:
sure, I can basically revert https://github.com/tmewett/BrogueCE/pull/87/commits/c0794992b728a7c02aecf0eecadbacd5e3b4b142 and put the 'install' back in the packaging directives. I was surprised there was no `install` directive, but I don't have a bone to pick.